views/layout.slim

```diff
@@ -44,11 +41,15 @@ html
       /! /.container
     script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"
     link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet" /
-    script src="assets/javascripts/bootstrap.min.js"
+    script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"
     script src="assets/javascripts/scripts.js"
```

Comment:
Recommend to try moving these script to bottom of a page. It will speed up initial page load dramatically.